util/db_test_util.h

```diff
@@ -442,9 +442,10 @@ class DBTestBase : public testing::Test {
     kFIFOCompaction = 26,
     kOptimizeFiltersForHits = 27,
     kRowCache = 28,
-    kLevelSubcompactions = 29,
-    kUniversalSubcompactions = 30,
-    kEnd = 29
+    kRecycleLogFiles = 29,
+    kLevelSubcompactions = 30,
+    kUniversalSubcompactions = 31,
+    kEnd = 30
```

Comment:
The change is very good. I know there are some coverage of multiple logs done in column_family_test. @igorcanadi , what tests in column_family_test do you think we should verify for recycled logs too?